.yamato/shared_triggers.yml

```diff
@@ -0,0 +1,45 @@
+{% metadata_file .yamato/project.metafile %}
+---
+
+# Run all relevant tasks when a pull request targets the develop / main or a sample branch
+pull_request_trigger:
+  name: Shared Pull Request Trigger (main, develop, & sample branches)

```

Comment:
I think that unless you call this "Pull Request Trigger (main, develop, & sample branches)" like it was in `_triggers.yml`, the 1st check of the pull request will never be triggered

<img width="1153" alt="image" src="https://github.com/Unity-Technologies/com.unity.multiplayer.samples.bitesize/assets/27730963/1fd04890-ef18-4b42-822f-92cc8648539e">

Reply:
Oh this helped. I changed the name and it cleared. Thank you @RikuTheFuffs !